src/main/java/org/sciborgs1155/robot/feeder/SimFeeder.java

```diff
@@ -0,0 +1,24 @@
+package org.sciborgs1155.robot.feeder;
+
+import static edu.wpi.first.units.Units.Seconds;
+import static org.sciborgs1155.robot.feeder.FeederConstants.*;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.system.plant.LinearSystemId;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import org.sciborgs1155.robot.Constants;
+
+public class SimFeeder implements FeederIO {
+
+  private final DCMotorSim sim =
+      new DCMotorSim(LinearSystemId.createDCMotorSystem(kV, kA), DCMotor.getNeoVortex(1), GEARING);
+
+  @Override
+  public void set(double power) {
```

Comment:
this `set` takes in power... but it then treats it as voltage

if we want to use closed loop control for the feeder, we can, but we aren't right now